packages/loader/container-loader/src/container.ts

```diff
@@ -664,6 +651,8 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     }
 
     public async request(path: IRequest): Promise<IResponse> {
+        await this.finishLoad(path);
```

Comment:
>await [](start = 8, length = 5)

would this need to be checked in more places? quorum? resume? Also is there a way to know a container isn't loaded